subprojects/launcher/src/main/java/org/gradle/launcher/cli/action/BuildActionSerializer.java

```diff
@@ -97,8 +97,6 @@ public void write(Encoder encoder, ExecuteBuildAction action) throws Exception {
             nullableFileSerializer.write(encoder, startParameter.getGradleHomeDir());
             nullableFileSerializer.write(encoder, startParameter.getProjectCacheDir());
             fileListSerializer.write(encoder, startParameter.getIncludedBuilds());
-            encoder.writeBoolean(startParameter.isUseEmptySettingsWithoutDeprecationWarning());
```

Comment:
This still needs to serialize the property on `StartParameterInternal`, see #15972